docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -14,10 +15,14 @@ If you don’t want to modify the behavior, there is normally no need to modify
 
 However, in some rare cases, configuration changes are required. These cases are described in the :ref:`Release Notes <releasenotes>`.
 
-With Bareos version 16.2.4 the default configuration uses the :ref:`section-SubdirectoryConfigurationScheme`. This scheme offers various improvements. However, if your are updating from earlier versions, your existing single configuration files (:file:`/etc/bareos/bareos-*.conf`) stay in place and are contentiously used by Bareos. The new default configuration resource files will also be installed (:file:`/etc/bareos/bareos-*.d/*/*.conf`). However,
-they will only be used, when the legacy configuration file does not exist.
+It is generally sufficient, to upgrade to the latest release,
+without having to install any intermediate releases
+(however, it is required to read the release notes of all intermediate releases).
+
+One exception is when using a |mysql| Bareos catalog. The |mysql| backend support has been removed in Bareos :sinceVersion:`21.0.0: MySQL backend removed`. Therefore you first have to upgrade to Bareos 20 and migrate the |mysql| into a |postgresql| Bareos Catalog, see :ref:`section-MigrationMysqlToPostgresql`.
+
+When updating from Bareos < 16.2.4, please read :ref:`section-UpdateToConfigurationSubdirectories`.
 
-See :ref:`section-UpdateToConfigurationSubdirectories` for details and how to migrate to :ref:`section-SubdirectoryConfigurationScheme`.
```

Comment:
We should keep this remark, many many many people have a mixed situation here.

Reply:
Do you propose to keep the full remark ("With Bareos version 16.2.4 the default  configuration uses ...") ?

Reply:
I believe it's nice to keep a cross link to both section, which contain also the previous informations.

Reply:
Readded old explaination.